client/containers/DashboardOverview/CollectionOverview/DashboardCollectionOverview.tsx

```diff
@@ -230,8 +238,21 @@ const DashboardCollectionOverview = (props: Props) => {
 		);
 	};
 
+	const renderBanner = (bannerText: String) => {
+		return (
+			<div style={{ background: lighterAccentColor }} className="banner">
+				<div className="banner-text">{bannerText}</div>
+			</div>
```

Comment:
It may be out of scope now since we're wrapping things up, but I think this is a good opportunity to create a reusable `<Banner>` component. It could optionally accept a `button` or `right` prop that would render something on the far right side of the banner. In this case, we could add a button that would navigate you to the Submissions tab with pressed.